src/components/Title.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Title = props =>  (
+        <div className="app__title">
```

Comment:
as I was mentioning in the task description, we need to have here an api to control the tag for the title. So, we should render properly the heading according to tag given (h1 or h2 or h3, etc).
Link is shared on slack ;) 

Plase, extract title styles to separate file also.